### DISCUSSION TITLE
奇怪的问题——引向外部站点的文章，是否应当连带抓取

### DISCUSSION THREAD

**Discussion by cronfox** [DIYgod/RSSHub#9421] [Q&A] [answered]
例子: https://luoqi.tiancity.com/homepage/article/Class_224_Time_1.html
其中有一部分是引到17173的，我正在开发相关路由规则，请问这种情况我是不抓取还是连带抓取?

**Comment by TonyRL** [comment, marked as answer]
If you want to see them, you can include them. If you don't want to see them, you can drop them.

**Reply by intzarbhatti444-ops** [reply]
yes

**Comment by intzarbhatti444-ops** [comment]
never behave
